requirements/2.7/requirements.txt

```diff
@@ -40,9 +40,6 @@ django-tagging==0.3.1
 pytz==2013.8
 
 # Cthulhu
-M2crypto==0.22.3
```

Comment:
I fail to understand your concern here. In the case where you use vagrant to set this up the base box must have salt to properly provision. In the case where you run vps_bootstrap.sh you end up with salt. Is there a third case?

Now that I say that I think it'd be nice to send vagrant down the same patch to so that we end up with the same version of salt in both cases.